django/core/management/templates.py

```diff
@@ -133,15 +132,15 @@ def handle(self, app_or_project, name, target=None, **options):
                 if filename.endswith(('.pyo', '.pyc', '.py.class')):
                     # Ignore some files as they cause various breakages.
                     continue
-                old_path = path.join(root, filename)
-                new_path = path.join(top_dir, relative_dir,
-                                     filename.replace(base_name, name))
+                old_path = os.path.join(root, filename)
+                new_path = os.path.join(top_dir, relative_dir,
+                                        filename.replace(base_name, name))
```

Comment:
Given that we're touching this, we should probably [use hanging indent](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).